lm_eval/api/utils.py

```diff
@@ -18,17 +18,17 @@ class ExitCodeError(Exception):
 # Reproducibility utils
 
 
-def get_seed() -> int:
-    """Returns a hard-coded global seed for reproducibility."""
+def get_default_seed() -> int:
```

Comment:
Why not use a constant?

Reply:
There are no constants in Python.

Reply:
`DEFAULT_SEED = 1234`?

Reply:
Sorry, I assumed you meant an immutable constant. I was hesitant to use that style because you miss out on docstrings but it's self-explanatory lol I'll update it.